.prettierignore

```diff
@@ -0,0 +1,10 @@
+public/content/languages
```

Comment:
I guess we can ignore the whole public folder for both prettier, eslint and stylelint if there is no original code there … (?)

Reply:
The format script should run in the `themes` folder that is located in `public`

Reply:
sorry. Yes we can ignore that one. Will update